src/main/java/pl/pjwstk/woloappapi/controller/OrganisationController.java

```diff
@@ -0,0 +1,41 @@
+package pl.pjwstk.woloappapi.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.pjwstk.woloappapi.model.Organisation;
+import pl.pjwstk.woloappapi.service.OrganisationService;
+
+import java.util.List;
+
```

Comment:
jak poprzednio - 1 pusta linia. Ważne jest to, aby kod był zawsze tak samo formatowany. Nieważne jak. Ważne, aby zawsze tak samo. Jeśli chcecie mieć tu 2 puste linie - ok, ale WSZĘDZIE. W każdej jednej klasie.